README.md

```diff
@@ -333,14 +331,17 @@ The paths and peer/org names contain variables that can be passed when starting
 * `ORG_NUMBER` is the number of the organization (1 for Org1, ..., 4 for Org4)
 * `NETWORK` is the name of the network (basic or multichannel)
 * `PEER_NUMBER` is the number of the peer (in basic network, it can be only 0, in multichannel, it can be 0 or 1)
-Thanks to these variables, we can run multiple instances at the same time with different configurations without rebuilding the agent.
-If we want to use the agent with another (custom) network, we have to modify the configuration according to the network's specifications. We can remove these variables and hardcode the names and paths, or use this example to come up with a similar solution.
+
+
+Thanks to these variables, it is possible to run multiple instances at the same time with different configurations without rebuilding the agent.
+
+To use the agent with another (custom) network, modify the configuration according to that network's specifications. 
 
 ### About indices
 
-We use 3 different indices per organization: one for blocks, one for transactions and one for single writes.  
-If we run multiple agents for peers in the same organization, they are goint to send their data to the same indices. We can then select the peer on the dashboards to view its data only.  
-If we run multiple instances for peers in different organizations, we are going to see the data of different organizations on different dashboards.  
+Three different Elasticsearch indices per Fabric organization are setup. One for blocks, one for transactions and one for single writes.  If wmultiple agents are run for peers in the same organization, they are goint to send their data to the same indices. You can then select the peer on the dashboards to view its data only.  
```

Comment:
"they are goint to" - please fix the typo. Otherwise, I like the changes, thanks for updating the readme.